docs/user-guide/zowe-secure-setup.md

```diff
@@ -0,0 +1,225 @@
+
+
+## High Availability
+
+To deploy Zowe high availability, you must set up the Parallel Sysplex® environment. A Parallel Sysplex is a collection of z/OS® systems that cooperatively use certain hardware and software components to achieve a high-availability workload processing environment.
+
+### Sysplex architecture and configuration
+
+Sysplex is required to make sure multiple Zowe instances can work together. Check [Configuring Sysplex for high availability](../user-guide/configure-sysplex) for more details.
+
+To enable high availability when Zowe runs in Sysplex, you need to meet the following requirements:
+
+- Zowe instance with all services should be installed on every LPAR.
+- Shared File system should be created between LPARs in Sysplex. See [How to share file systems in a Sysplex](https://www.ibm.com/docs/en/zos/2.4.0?topic=planning-sharing-file-systems-in-sysplex). 
+- z/OSMF High Availability mode should be configured. See [Configuring z/OSMF high availability in Sysplex](../user-guide/systemrequirements-zosmf-ha).
+
+Instance on every LPAR is started.
+
+**Configuration**
+
+The configuration for the specific instance is composed of the defaults in the main section and the overrides in the `haInstances` section.
+
+In this section, <ha-instance> represents any Zowe high availability instance ID. Every instance has internal id and a section with overrides compared to the main configuration in the beginning of the `zowe.yaml` file. Check [Zowe YAML configuration reference](../appendix/zowe-yaml-configuration/#yaml-configurations---hainstances) for details.
+
+### Caching service setup and configuration
+
+Zowe uses the Caching Service to centralize the state data persistent in high availability (HA) mode. It can be used to share information between services.
+
+If you are runnning the caching service on z/OS, there are three storage methods with their own characteristics:
+
+- [VSAM](../user-guide/configure-caching-service-ha)
+  - Known by zOS engineers.
+  - Slow.
+- [Redis](../extend/extend-apiml/api-mediation-redis#redis-configuration)
+  - Needs to run in Distributed world separately.
+  - Good for Kubernetes deployment.
+- [Infinispan (*recommended*)](../extend/extend-apiml/api-mediation-infinispan#infinispan-configuration)
+  - Part of the Caching service.
+  - Doesn’t need separate processes.
+  - Highly performant.
+
+## Observation
+
+It is essential to have observation in the whole secure operation process. The service availability is visible to everyone. It’s easy to find performance issues with southbound services.
+
+To do so, Zowe can easy integrate with the Alerting and Monitoring enterprise services such as ELK, Splunk, Grafana, etc and integrate the data as part of the Zero Trust Architecture approach with, for example SIEM (Security Incident Event Management). 
+
+### Use Audit Log to observe
```

Comment:
Remove the Audit Log please as it's vendor specific until we figure out whether and how we will open source the solution.

Reply:
Thanks for your comment Jakub! just want to double confirm if the whole `Observation` section should be removed as well.

Reply:
No, the rest is still valid.